codec/src/main/java/io/netty/handler/codec/compression/BrotliEncoder.java

```diff
@@ -0,0 +1,40 @@
+package io.netty.handler.codec.compression;
+
+import com.nixxcode.jvmbrotli.enc.BrotliOutputStream;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufOutputStream;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.MessageToByteEncoder;
+
+public class BrotliEncoder extends MessageToByteEncoder<ByteBuf> {
```

Comment:
As a feature with a native lib behind it, how do you want to deal with library loading issues?

From https://github.com/nixxcode/jvm-brotli/blob/master/README.md

You MUST call the following method at least once during your application's runtime if you wish to use Brotli:
BrotliLoader.isBrotliAvailable();

Reply:
We can place it under the static block of the Encoder or Decoder